### PR TITLE
3/16 changes

### PR DESCRIPTION
**Pull request by TonyBaston**
This has a decent amount of changes, but it is tested.

### CODE REVIEW

**Comment by TonyBaston** [comment]
It says there are conflicts, but I don't quite see what it means by them, as they are just changes.

File: ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Hood.java [hunk 1]
```diff
@@ -100,27 +99,34 @@ public void updateSmartDashboard() {
   public void init(){}
   public void periodic() {
     if (m_joystick.getPOV() == 90) {
```
Comment (crschardt) [inline ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Hood.java:101]: @TonyBaston - This only works for the XBox joystick. Are you planning to abstract this too? If not, is this required for operating the bot, or just for testing?

**Review by crschardt — APPROVED** [review]

**Comment by crschardt** [comment]
> 

The conflicts indicate that main had changes pushed to it that weren't reflected in the new branch. This is probably because the branch was created from an older version of main. I've reviewed the conflicts and addressed them as best I could, but there might be problems with the code that need fixing after the merge.

**Review by crschardt — APPROVED** [review]

File: ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Hood.java (hunk 1 above)
Comment (TonyBaston) [reply ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Hood.java:101]: This was just for testing

**Pull request merged**